README.md

```diff
@@ -1,18 +1,17 @@
 # Mattermost Handbook
 
-The purpose of this handbook is to _**iteratively increase clarity**_ for Mattermost staff and community co-building the future of our company and our offerings.
+The Mattermost Handbook is a continual work-in-progress for _Mattermost staff_--people who are paid to work on our open source software and to help run the company behind our software. The purpose of this handbook is to _**iteratively increase clarity**_ for Mattermost staff and community co-building the future of our company and our offerings.
```

Comment:
```suggestion
The Mattermost Handbook is a continual work-in-progress for Mattermost staff (people who are paid to work on our open source software) and contributors (our indispensable community members). We use it to document the processes that underpin our company behind our software. Its purpose is to **iteratively increase clarity** for Mattermost staff and community co-building the future of our company and our offerings, and provide a single-source of truth.
```

Reply:
I would adjust the sentence to read `... for Mattermost staff (people who are paid to work on our open source software), contributor community and end user community.

(since not everyone in our community are contributors, but pretty much everyone are 'users' - whether users of our product, our APIs, or perhaps our handbook :) )